test/Datadog.Trace.Tests/Configuration/ConfigurationSourceTests.cs

```diff
@@ -70,9 +74,9 @@ public static IEnumerable<object[]> GetTestData()
 
             yield return new object[] { ConfigurationKeys.DisabledIntegrations, "integration1;integration2", CreateFunc(s => s.DisabledIntegrationNames.Count), 2 };
 
-            yield return new object[] { ConfigurationKeys.GlobalTags, "k1:v1, k2:v2", CreateFunc(s => s.GlobalTags.Count), 2 };
-            yield return new object[] { ConfigurationKeys.GlobalTags, "k1:,:,:v2,k3:v3", CreateFunc(s => s.GlobalTags.Count), 1 };
-            yield return new object[] { "DD_TRACE_GLOBAL_TAGS", "k1:v1, k2:v2", CreateFunc(s => s.GlobalTags.Count), 2 };
+            yield return new object[] { ConfigurationKeys.GlobalTags, "k1:v1, k2:v2", CreateFunc(s => s.GlobalTags), TagsK1V1K2V2 };
+            yield return new object[] { ConfigurationKeys.GlobalTags, "k1:,k3,:,:v2,k2:v2", CreateFunc(s => s.GlobalTags), TagsK2V2 };
```

Comment:
For readability, consider using meaningful strings for the broken tags:
```suggestion
            yield return new object[] { ConfigurationKeys.GlobalTags, "keyonly:,nocolon,:,:valueonly,k2:v2", CreateFunc(s => s.GlobalTags), TagsK2V2 };
```

Reply:
Thanks for the suggestion, applied in 3be3025d1ae6cd911e9b3d45d87669396a023d49